### ISSUE TITLE
RuntimeError: File loading is not yet supported on Windows

### ISSUE THREAD

**Issue by gkapellmann** [google/mediapipe#4284]
### Have I written custom code (as opposed to using a stock example script provided in MediaPipe)

Yes

### OS Platform and Distribution

Windows 11

### Mobile device if the issue happens on mobile device

Desktop

### Programming Language and version

Python 3.9

### MediaPipe version

0.9.1

### Solution

Hand Gesture

### Describe the actual behavior

```shell
It send the error as described in the title.

"File loading is not yet supported on Windows"
This when in this line:

'vision.GestureRecognizer.create_from_options(options)'

Is there a workaround for this? Or is it literally no way to make this work on Windows yet?
```

### Describe the expected behaviour

```shell
Basically to load the recognizer.
```

### Standalone code/steps you may have used to try to get what you need

```shell
base_options = python.BaseOptions(model_asset_path='gesture_recognizer.task')
options = vision.GestureRecognizerOptions(base_options=base_options)
self.recognizer = vision.GestureRecognizer.create_from_options(options)
```

**Comment by kuaashish** [comment]
Hello @gkapellmann,
Thank you for your interest in the MediaPipe solutions. This issue already known to us and already been fixed in master that will be part of the next release, which is coming within the next couple of days.

**Comment by pointcloudAI** [comment]
> 

Would you please tell us which submit can fix this problem ? So we can try to fix it locally.
